src/library/columncache.cpp

```diff
@@ -90,27 +91,33 @@ void ColumnCache::setColumns(const QStringList& columns) {
         m_columnIndexByEnum[it.key()] = fieldIndex(it.value());
     }
 
-    const QString sortInt("cast(%1 as integer)");
-    const QString sortNoCase("lower(%1)");
+    const QString sortInt = QStringLiteral("cast(%1 as integer)");
```

Comment:
Move into anonymous namespace?

Reply:
Why? This is only used once.

Reply:
No, the string are recreated on every setColumns() call. I will move them.